### DISCUSSION TITLE
Full list of timezone values

### DISCUSSION THREAD

**Discussion by soyeric128** [databendlabs/databend#7256] [Q&A] [answered]
Hi there, 

I got a question regarding the document: https://databend.rs/doc/reference/functions/context-functions/timezone

In the example, the timezone is set to China Standard Time by the following code:

`SET timezone='Asia/Shanghai';`

What's the full list of the available values that can be assigned to the timezone setting? I didn't find it in the documentation. 

Thanks,

**Comment by bohutang** [comment, marked as answer]
Databend using chrono timezone enum:
 https://docs.rs/chrono-tz/latest/chrono_tz/enum.Tz.html
